server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -4922,7 +4922,9 @@ public boolean releasePublicIpRange(final ReleasePublicIpRangeCmd cmd) {
     public boolean releasePublicIpRange(final long vlanDbId, final long userId, final Account caller) {
         VlanVO vlan = _vlanDao.findById(vlanDbId);
         if(vlan == null) {
-            s_logger.warn("VLAN information for Account '" + caller + "', User '" + userId + "' VLAN '" + vlanDbId + "' is null. This is NPE situation.");
+            // Nothing to do if vlan can't be found
+            s_logger.warn("VLAN information for Account '" + caller + "', User '" + userId + "' VLAN '" + vlanDbId + "' is null.");
+            return true;
```

Comment:
@ravening can you address this ^^^

Reply:
@DaanHoogland yes, what @GabrielBrascher  says is correct.. since vlan cant be found in db, just return true rather than proceeding further.. i will make the required changes

Reply:
Hi @ravening Can you address the required changes as suggested.

Reply:
@sureshanaparti made the change